prime-dotnet-webapi/ViewModels/Emails/PaperEnrolleeSubmissionEmailViewModel.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace Prime.ViewModels.Emails
+{
+    public class PaperEnrolleeSubmissionEmailViewModel
+    {
+        public string GPID { get; set; }
+
+        public PaperEnrolleeSubmissionEmailViewModel(string gpid)
```

Comment:
By default when creating a class it has a default constructor that you can access but doing:

```
new PaperEnrolleeSubmissionEmailViewModel
{
    GPID = value;
};
```

So there is not a need to define this constructor and you can use the above to call in your code.